.github/workflows/delete-ns.yml

```diff
@@ -0,0 +1,36 @@
+name: delete-ns
```

Comment:
May I suggest a more descriptive name for this file and workflow?

`delete-namespace.yml` for the workflow, and for the name:
```yml
name: delete-namespace
```

Reply:
Done